src/cli/helpers.ts

```diff
@@ -17,16 +17,26 @@ import { builtinParameterTypes } from '../support_code_library_builder'
 import { version } from '../version'
 
 interface IParseGherkinMessageStreamRequest {
-  logger: Console
+  cwd?: string
```

Comment:
I cannot find any usage for `cwd`. Is it required?

Reply:
It's not, but I wanted to maintain the signature as of 7.x so it's not a breaking change. I just made it optional which seemed like a good middle ground.